src/main/java/app/gpx_animator/Renderer.java

```diff
@@ -232,13 +248,14 @@ private void drawBackground(final RenderingContext rc, final Integer zoom, final
             ga.setColor(backgroundColor);
             ga.fillRect(0, 0, bi.getWidth(), bi.getHeight());
         } else {
-            MapUtil.drawMap(bi, cfg.getTmsUrlTemplate(), cfg.getBackgroundMapVisibility(), zoom, minX, maxX, minY, maxY, rc);
+            Map.drawMap(bi, cfg.getTmsUrlTemplate(), cfg.getBackgroundMapVisibility(), zoom, minX, maxX, minY, maxY, rc);
```

Comment:
I renamed this class one or two days ago. After you pull from upstream, this change should be gone.